zerver/models.py

```diff
@@ -289,6 +289,11 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     # at `foo.example.com`.
     string_id = models.CharField(max_length=MAX_REALM_SUBDOMAIN_LENGTH, unique=True)
 
+    # uuid and a secret for the sake of per-realm authentication with the push notification
+    # bouncer.
+    uuid = models.UUIDField(default=uuid4, unique=True)
+    uuid_owner_secret = models.CharField(max_length=32, default=generate_api_key, unique=True)
```

Comment:
Is `generate_api_key` a good function for generating this secret? It uses `secrets` under the hood, which is our standard as discussed. We could also do a direct `secrets.token_hex` call or something

Reply:
Yeah, I think that seems fine; this should have similar requirements to our normal API keys.